goaviatrix/gateway.go

```diff
@@ -619,3 +621,63 @@ func (c *Client) DisableVpcDnsServer(gateway *Gateway) error {
 	}
 	return nil
 }
+
+func (c *Client) EnableVpnNat(gateway *Gateway) error {
+	Url, err := url.Parse(c.baseURL)
+	if err != nil {
+		return errors.New(("url Parsing failed for enable_nat_on_vpn_gateway") + err.Error())
+	}
+	enableVpnNat := url.Values{}
+	enableVpnNat.Add("CID", c.CID)
+	enableVpnNat.Add("action", "enable_nat_on_vpn_gateway")
+	enableVpnNat.Add("vpc_id", gateway.VpcID)
+	enableVpnNat.Add("lb_or_gateway_name", gateway.GwName)
```

Comment:
similar to split tunnel. This needs to be elb_name if gateway has elb enabled. It is gw_name only if ELB is disabled